src/constants/imageConstants.ts

```diff
@@ -0,0 +1,14 @@
+export const IMAGES = {
+  logoImg: require('@assets/logo.png'),
+  titleImg: require('@assets/taskWave.png'),
+  formTop: require('@assets/formTop.png'),
+  noListFound: require('@assets/noListFound.jpeg'),
+  taskWaveTitle: require('@assets/taskwaveTitle.png'),
+  check: require('@assets/check.png'),
+  checklist: require('@assets/checklist.png'),
+  paper: require('@assets/paper.png'),
+  prioritize: require('@assets/prioritize.png'),
+  target: require('@assets/target.png'),
+  tasks: require('@assets/tasks.png'),
+  todoList: require('@assets/to-do-list.png'),
```

Comment:
`to-do-list.png`  - as I have commented earlier same logic goes for image file names